go/serialization.go

```diff
@@ -126,10 +130,10 @@ func readASN1CertList(r io.Reader, totalLenBytes int, elementLenBytes int) ([]AS
 // Returns a non-nil error if there was a problem.
 func ReadTimestampedEntryInto(r io.Reader, t *TimestampedEntry) error {
 	var err error
```

Comment:
That line might not be needed anymore, with your other changes? (I like your way better!)